packages/lisk-api-client/.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-	"extends": ["../../.eslintrc.json"]
+	"extends": ["../../.eslintrc.json"],
+	"globals": {
+		"mocha": true,
+		"document": true
+	}
```

Comment:
These globals don't belong in the root of the package. They should be in `.eslintrc.json` files in the relevant subdirectory/ies. Also these are better set via the `mocha` and `browser` `env` settings.